### DISCUSSION TITLE
Model Revisions

### DISCUSSION THREAD

**Discussion by philharmonie** [filamentphp/filament#8272] [Ideas]
Instead of editing a model, the edit would create a new row (revision) with an incremented revision. This would create an edit history of a model. The feature would take care about revision deletion in correct order (revision by revision) until it is entirely deleting the row.

**Comment by danharrin** [comment]
There are Laravel packages you can use to do this, it would never be a Filament feature

**Reply by philharmonie** [reply]
Why not? Having revisions of models would add a beneficial level of administration

**Reply by danharrin** [reply]
Because we are a UI framework primarily, that is too opinionated on the database level, and already achievable without a core feature
